src/screens/home/home.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { ScreenContainer } from '../../components/screen-container/screen-container';
+import { network } from '../../config/general';
+import { loadAssetsActions } from '../../store/assets/assets-actions';
+import { useAssetsSelector } from '../../store/assets/assets-selectors';
+import { useFirstAccountSelector } from '../../store/wallet/wallet-selectors';
+import { HomeStyles } from './home.styles';
+
+export const Home = () => {
+  const firstAccount = useFirstAccountSelector();
+  const assets = useAssetsSelector();
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(loadAssetsActions.submit({ network, address: firstAccount.publicKeyHash, size: 10, offset: 0 }));
```

Comment:
`network`, `size` and `offset` are not configured right now.  
Just hardcode them inside `requestGetAssets` and `loadAssetsActions.submit` would accept string payload (publicKeyHash)